### DISCUSSION TITLE
Panning in the volume column

### DISCUSSION THREAD

**Discussion by bootsymcb** [schismtracker/schismtracker#469] [Q&A] [answered]
Hiya, the manual mentions that you can control panning in the volume column but as far as I can tell it neglects to mention how.

In this particular instance I'm already pitch-shifting in the effects column so I need another way to set the panning. Thanks.

**Comment by sagamusix** [comment, marked as answer]
From the Impulse Tracker manual:

>          3) The third column contains the volume or panning of the note.
>            To toggle between entry of volumes and panning, press '`'

That should be the key left of the `1` key. When pressing it, you should see "Panning control set" or "Volume control set" respectively, and this will affect all volume column entries done afterwards.

**Comment by bootsymcb** [comment]
I dunno how I overlooked that, thanks.
